website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Project dependencies"
+id: project-dependencies
+sidebar_label: "Project dependencies"
+description: "Reference public models across dbt projects"
+---
+
```

Comment:
TODO: Add a note to this page about project-level cycle detection (https://github.com/dbt-labs/dbt-core/issues/7468)